packages/rebound-router/lib/rebound-router.js

```diff
@@ -19,6 +19,19 @@ var SUCCESS = 'success';
 var ERROR = 'error';
 var LOADING = 'loading';
 
+// If the string passed to this function is a valid query string, split its values
+// out into a hash and return the object.
+function parseQueryString(str){
+  var res = {};
+  if(!str || str.indexOf('=') == -1) return str;
+  str = str.split('&');
+  str.forEach(function(item){
+    item = item.split('=');
+    res[item[0]] = item[1];
+  });
+  return res;
+};
```

Comment:
What about a common pattern such as `?name=james&name=adam`

For more, check out [this SO Q&A](http://stackoverflow.com/questions/1746507/authoritative-position-of-duplicate-http-get-query-keys).